docs/docusaurus/versioned_docs/version-0.18/cloud/deploy_gx_agent.md

```diff
@@ -179,7 +179,7 @@ You can deploy the GX Agent in any environment in which you create Kubernetes cl
     spec:
       containers:
        name: gx-agent
-        image: greatexpectations/agent:latest
+        image: greatexpectations/agent:stable
```

Comment:
The docker command should also be updated.

Reply:
Thanks :) updated